.travis.yml

```diff
@@ -54,7 +54,7 @@ env:
   - REPO_TYPE=nix
   - REPO_TYPE=dockerfile
   - REPO_TYPE=external/*
-  - REPO_TYPE=**/*.py
```

Comment:
A simpler fix for me would be to move all the unittests into a `tests/unit` directory, then here would be `REPO_TYPE=unit`

Reply:
This would allow us to organise the unit tests into sub-directories again?